master/pkg/model/job.go

```diff
@@ -0,0 +1,40 @@
+package model
+
+import (
+	"github.com/google/uuid"
+)
+
+// JobID is the unique ID of a job among all jobs.
+type JobID string
+
+func (id JobID) String() string {
+	return string(id) // FIXME does this add any value? is this a common interface in go?
```

Comment:
not really. `String() string` is the method to impl `Stringer`, so anything that would use Stringer if it exists (fmt.Sprintf/fmt.Printf internally) will call it, but since the type aliases string it has the same behavior anyway (but it you wanted it to print, say, "Job: :id" as it's default %s repr, then it could be useful